MapShare/MapShare/Helper Files/UIElements.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  UIElements.swift
+//  MapShare
+//
+//  Created by iMac Pro on 5/5/23.
+//
+
+import UIKit
+
+struct UIElements {
+    
+    static func
```

Comment:
What's going on here? Did this get cut off?

Reply:
I deleted that file. Nothing should be there.